site/content/docs/manifest/worker-service.en.md

```diff
@@ -55,7 +58,7 @@ The architecture type for your service. [Worker Services](../concepts/services.e
 
 <div class="separator"></div>
 
-<a id="subscribe" href="#subscribe" class="field">`subscribe`</a> <span class="type">Map</span>
+<a id="subscribe" href="#subscribe" class="field">`subscribe`</a> <span class="type">Map</span>  
```

Comment:
Why do we need the trailing space 🤔

Reply:
It's how markdown does new lines 😂 https://stackoverflow.com/a/33191810/1201381

Otherwise, the text appears right next to `subscribe`